src/assets/js/archive.js

```diff
@@ -5,62 +5,109 @@ const releasesToShow = 99999;
 function fetchFlutterReleases(os, callback, errorCallback) {
   // OS: windows, macos, linux
   const url = `https://storage.googleapis.com/flutter_infra_release/releases/releases_${os}.json`;
-  $.ajax({
-    type: 'GET',
-    url: url,
-    dataType: 'json',
-    success: function (data) { callback(data, os); },
-    error: function (xhr, textStatus, errorThrown) {
-      if (errorCallback) errorCallback(os);
-    }
-  })
+  fetch(url, { type: 'GET'})
+    .then(response => response.json())
+    .then(data => callback(data, os))
+    .catch(_ => { if (errorCallback) errorCallback(os) })
```

Comment:
I personally find expanding this a bit easier to read since it's less condensed. Also can add semicolons to be consistent with elsewhere.

```suggestion
    .catch(_ => {
      if (errorCallback) errorCallback(os);
    });
```

Reply:
I'll make this, I'll add semicolon and also brackets in a if with only one statement, for readbility.

Reply:
Solved with this commit [#c522b0](https://github.com/flutter/website/pull/9022/commits/c522b0bee24cd8f78185e2b5abea12991855a551)